.cargo/config

```diff
@@ -0,0 +1,3 @@
+# Accessed by wasm-bindgen when testing for the wasm target
+[target.wasm32-unknown-unknown]
+runner = 'wasm-bindgen-test-runner'
```

Comment:
could you try adding the necessary tooling and commands to make this run with wasm ?

Reply:
I can. There is a small issue though. I can run `cargo test` in `tfhe`, which will compile `concrete-fft` properly. When running it in just the `concrete-fft` though, there seems to be a build script that runs before compiling the source code. And that script seems to attempt to check for something in my hardware that isn't available.

I have a Linux machine running in an AMD CPU. I think that the build script is expecting me to have a Mac instead.

I could create an issue to talk about it. It is a bit hard for me to run tests directly on this crate on the current situation.

Don't have much time to focus on this issue too. If you are able to implement this fix yourselves and I'm taking too long to push it, don't let me hold you back.

If we can fix the build bug that happens when compiling this crate locally, I can add the wasm-pack testing tools.

Reply:
do you know which build script is causing issues ? 🤔

Reply:
This is the output whenever I run `cargo test --release`.

![Bugconcretefft](https://github.com/zama-ai/concrete-fft/assets/61277568/0c3804a8-7ea0-4647-bcf7-ce60d0fc37d4)

Reply:
`sudo apt-get install m4` on Ubuntu apparently

looks like some C library is compiled and use with FFI by some crate 🤔

Reply:
I think it is weird because there is no need to do that when compiling tfhe-rs. It is probably something with the build script. But I didn't have much time to look into it.

Reply:
We don't have a build script for concrete-fft so it must be some web/wasm/js dependency thing ?

Reply:
I fixed it. What is "rug" used for in this crate? Those checks are made by the crate `gmp-mpfr-sys`, which is used by `rug`. I just removed `rug` from Cargo.toml and all the tests worked.

Reply:
https://crates.io/crates/rug

Reply:
looks to be a wrapper for gnu multi precision arbitrary precision arithmetic code, it's weird we have that dependency if we don't use it 🤔

Reply:
we use it for f128 so it would mean we would not be able to test f128 ffts for wasm, we need that dependency though

Reply:
I don't know if dev dependencies can be optional

Reply:
`rug` is used for generating random integers for tests in "fft128" feature. It is not used in the implementation. I think I will just comment those tests since we are just fixing wasm implementation. Will include tests for wasm-bindgen runtime too.

Reply:
I will prepare a new pr with the suggested changes. Just wanted to keep you updated on what I found.

Reply:
ok thanks :)

Reply:
Here #32